kraken-build/src/kraken/std/cargo/__init__.py

```diff
@@ -363,14 +363,16 @@ def cargo_test(
     project: Project | None = None,
     features: list[str] | None = None,
     depends_on: Sequence[Task] = (),
+    workspace: bool | None = None,
```

Comment:
Shouldn't it default to `True`?

Reply:
That'd be a semantic break here, and inconsistent with `cargo_build()`. We can set our opinionated default internally.